src/templates/header/index.hbs

```diff
@@ -25,7 +25,7 @@
                     <li data-auth-wrapper class="c-nav-list-item has-sublist {{#unless user.isAuthenticated}}is-hidden{{/unless}}" data-test-id="navList" tabindex="0">
                         <p class="c-nav-list-text">
                             {{> nav-icon }}
-                            <span data-name class="c-nav-list-text-sub">{{#if user.isAuthenticated}}{{user.friendlyName}}{{/if}}</span>
+                            <span data-name class="c-nav-list-text-sub"{{#if user.isAuthenticated}} title="{{user.friendlyName}}{{/if}}">{{#if user.isAuthenticated}}{{user.friendlyName}}{{/if}}</span>
```

Comment:
Should that last `"` be inside the `if` block?

Reply:
No, the quotes are for the class attribute. The title attribute is a new attribute that needs to be inside the conditional.

Reply:
From what I'm reading, I think it will be rendered as this if `user.isAuthenticated` is `true`:

`<span data-name class="c-nav-list-text-sub" title="name">`

which is all good and correct, but if `user.isAuthenticated` is `false`, it will be

`<span data-name class="c-nav-list-text-sub"">`

(Notice the double close quotes at the end, as the closing `"` for the title attribute isn't inside the conditional)

Reply:
Well spotted. Fixed